FastTimeSeries/src/models.jl

```diff
@@ -0,0 +1,27 @@
+
+"""
+blockmodel(inblock::TimeSeriesRow, outblock::OneHotTensor{0}, backbone)
+
+Construct a model for time-series classification.
+"""
+function blockmodel(inblock::TimeSeriesRow,
+                outblock::OneHotTensor{0}, 
+                backbone)
+    data   = rand(Float32, inblock.nfeatures, 32, inblock.obslength)
```

Comment:
Should be using [`Flux.outputsize`](https://fluxml.ai/Flux.jl/stable/utilities/#Flux.outputsize) (I think we talked about this in a meeting). Changes required should be minimal :)

Reply:
I tried doing that, but seems to give an error. Will try to debug it later.
https://pastebin.com/mB7aYvdD

Reply:
I see, this is https://github.com/FluxML/Flux.jl/pull/1755#issue-1035506488. Can you do the following then?
- Use `zeros` or `ones` instead of `rand`.
- Pass a batch size of 1 instead of 32 and seq length of 1 instead of `inblock.obslength`. This will save some time and you don't need it because you only care about the first output dimension.
- `reset!` the backbone after you calculate `output`.

Reply:
Yes, sounds great.